### PR TITLE
Google eol messaging

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) adds Google Shopping ads End of Life information.

## Affected documentation pages

- https://docs.magento.com/m2/ce/user_guide/sales-channels/google-ads/google-eol.html
- https://docs.magento.com/m2/ce/user_guide/sales-channels/google-ads/google-ad-channel.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional Info

1 new topic.  header added to all google topics.

### CODE REVIEW

File: src/sales-channels/google-ads/google-eol.md [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+
+    - Update product feed data directly from your Google Merchant Center account. See the [Google Onboarding Guide](https://support.google.com/merchants/answer/7439058?hl=en).
+
+    - Visit the [Magento Marketplace](https://marketplace.magento.com/) to find a Google product feed extension that meets your needs.
+
+1. What will happen to the catalog I am currently syncing back to GMC?
+
+    The catalog feed synced with the extension will stop updating in Google Merchant Center on April 28, 2020. For more information on setting up a new product feed, see the [Google Onboarding Guide](https://support.google.com/merchants/answer/7439058?hl=en).
+
+1. What actions and when do I need to take prior to disable date to minimize any disruption?
```
Comment (leslietilling) [inline src/sales-channels/google-ads/google-eol.md]:
```suggestion
1. What actions and when do I need to take prior to the disable date to minimize any disruption?
```

File: src/sales-channels/google-ads/google-eol.md [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+
+    We recommend you explore the following two options:
+
+    - Update product feed data directly from your Google Merchant Center account. See the [Google Onboarding Guide](https://support.google.com/merchants/answer/7439058?hl=en).
+
+    - Visit the [Magento Marketplace](https://marketplace.magento.com/) to find a Google product feed extension that meets your needs.
+
+1. What will happen to the catalog I am currently syncing back to GMC?
+
+    The catalog feed synced with the extension will stop updating in Google Merchant Center on April 28, 2020. For more information on setting up a new product feed, see the [Google Onboarding Guide](https://support.google.com/merchants/answer/7439058?hl=en).
```
Comment (leslietilling) [inline src/sales-channels/google-ads/google-eol.md]:
```suggestion
    The catalog feed synced with the extension will stop updating in Google Merchant Center on April 28, 2020. For more information about setting up a new product feed, see the [Google Onboarding Guide](https://support.google.com/merchants/answer/7439058?hl=en).
```

**Comment by hickskenh** [comment]
running tests

File: src/_includes/layout/page-header.html [hunk 3]
```diff
@@ -7,5 +7,11 @@
   </div>
 {% endif %}
 
+{% if page.tag == "google-shopping" %}
+  <div class="message-banner">
+    <b>Google Shopping ads Channel will be disabled on April 28, 2020. See <a href="https://docs.magedevteam.com/373/m2/ce/user_guide/sales-channels/google-ads/google-eol.html">End of Life</a> for more information.</b>
```
Comment (leslietilling) [inline src/_includes/layout/page-header.html]: Update link from magedevteam

**Comment by dshevtsov** [comment]
please consider to use the solution without need to add a tag inside each file in a directory

```
{% if page.url contains "sales-channels/google-ads/" %}
  <div class="message-banner">
    <b>Google Shopping ads Channel will be disabled on April 28, 2020. See <a href="https://docs.magedevteam.com/373/m2/ce/user_guide/sales-channels/google-ads/google-eol.html">End of Life</a> for more information.</b>
  </div>
{% endif %}
```

File: src/_includes/layout/page-header.html (hunk 3 above)
Comment (hickskenh) [reply src/_includes/layout/page-header.html]: fixed

**Comment by hickskenh** [comment]
running tests

**Comment by leslietilling** [comment]
I would do a squash for merging this

**Pull request merged**